packages/groqd-playground/tsup.config.ts

```diff
@@ -0,0 +1,10 @@
+import { defineConfig } from "tsup";
+
+export default defineConfig({
+	entry: ["src/index.ts"],
```

Comment:
Tiny thing but spacing seems off here.

Reply:
oh yeah, you're right – I wonder why ESLint didn't pick this up 🤔 I'll take a peak.

Reply:
`eslint packages/**/src --quiet` there it is, only checking against `src` files, whoops.